extensions/wikia/RevisionUpvotes/sql/schema.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS `upvote_revisions` (
+	`upvote_id` INT unsigned NOT NULL AUTO_INCREMENT,
+	`wiki_id` INT unsigned NOT NULL,
+	`revision_id` INT unsigned NOT NULL,
+	`page_id` INT unsigned NOT NULL,
+	`user_id` INT unsigned NOT NULL,
+	PRIMARY KEY (`upvote_id`),
+	UNIQUE KEY `upvote_revision_unique` (`wiki_id`, `revision_id`)
+) ENGINE = INNODB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+CREATE INDEX `upvote_revision_idx` ON `upvote_revisions` (`wiki_id`, `revision_id`);
```

Comment:
This is unneeded as you're already making a unique key above for the same columns. So, either remove this or remove the unique key and make this `CREATE UNIQUE INDEX`.